src/Robot.elm

```diff
@@ -0,0 +1,89 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Browser.Events
+import Html exposing (Html, button, div, text)
+import Html.Attributes as Attributes exposing (style)
+import Html.Events exposing (onClick)
+import Json.Decode
+
+main =
+    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }
+
+
+type alias Model =
+    { xblue : Int, yblue : Int , xred : Int, yred : Int}
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
+    ( { xblue = 50, yblue = 50 ,xred = 10, yred = 10}, Cmd.none )
+
+
+type Msg
+    = KeyDown Int
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    Browser.Events.onKeyDown (Json.Decode.map KeyDown Html.Events.keyCode)

```

Comment:
Use `key` rather than `keyCode`.
`keyCode` is about to be deprecated. and `key` returns you the name of the button, which is much nicer to use than its number.

Reply:
This is very important @yehonatancohen